.github/workflows/test-action.yml

```
@@ -0,0 +1,118 @@
+name: "Test GitHub Action"
+description: "Tests the GitHub Action as if it were used in a workflow by providing different inputs and verifying the outputs."
```

actionlint (1.7.12)

[error] unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)